frontend/src/app/core/services/project.service.ts

```diff
@@ -51,4 +53,12 @@ export class ProjectService {
     public getProjectUsers(projectId: number): Observable<UserDto[]> {
         return this.httpService.getRequest<UserDto[]>(`${this.projectsApiUrl}/team/${projectId}`);
     }
+
+    public getCurrentProjectId() {
+        return this.currentProjectId;
+    }
+
+    public setCurrentProjectId(id: number) {
+        this.currentProjectId = id;
+    }
```

Comment:
these getter and setter do nothing. Consider removing them. And by the way, TS has built-in getters and setters as in C#